### PR TITLE
Rectified the grammatical error and sentences in Virtualization.md 

### PR DESCRIPTION
**Pull request by Debaditya-Som**
### Description of the changes
This Pull Request aims to rectify the the grammatical errors present in the documentation and also changes the sentences that did not make sense before. Ref: #2403 

### Related issue number or link : 
Closes issue : #2403 

**Benefits of the PR**

1. **Improved Clarity**: Correcting the grammatical errors in the Bengali translation will make the content more clear and comprehensible to Bengali-speaking users who visit the CNCF Glossary to understand technical terms.

2. **Enhanced Accessibility**: By providing a more accurate translation, this PR will improve the accessibility of technical information for the Bengali-speaking audience, ensuring that they can access and use the content effectively.

3. **Quality Assurance**: Addressing these grammatical errors reflects the commitment to maintaining high-quality translations on the CNCF Glossary, which enhances the overall credibility of the resource.

**Additional Information**

- Language: Bengali

**Note:** 
_If you have proficiency in Bengali and can suggest improvements beyond the identified corrections, your input would be highly valuable._

### Checklist 

- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by Debaditya-Som** [comment]
@sajibAdhi @Arindam200 @Imtiaz1234 @mitul3737 
Could you kindly review this PR.
This fixes issue: #2404

File: content/bn/virtualization.md [hunk 1]
```diff
@@ -5,27 +5,28 @@ category: প্রযুক্তি
...
+শারীরিক হার্ডওয়্যার ব্যবহারের উন্নতি সহ বেশ কয়েকটি সমস্যার সমাধান করে।
 
-## এটা কিভাবে সাহায্য করে
 
-ভার্চুয়াল মেশিনে চলমান অ্যাপগুলির কোন সচেতনতা নেই যে তারা একটি শারীরিক কম্পিউটার ভাগ করছে।
-ভার্চুয়ালাইজেশন ডেটাসেন্টার ব্যবহারকারীদের মিনিটের মধ্যে একটি নতুন "কম্পিউটার" (ওরফে একটি ভিএম) স্পিন করতে দেয়
-একটি ডেটাসেন্টারে একটি নতুন কম্পিউটার যোগ করার শারীরিক সীমাবদ্ধতার বিষয়ে চিন্তা না করে।
+## এটা কিভাবে সাহায্য করে ?
+
+ভার্চুয়াল মেশিনে চলমান অ্যাপগুলির কোন সচেতনতা নেই, যে তারা একটি শারীরিক কম্পিউটার ভাগ করছে।
```
Comment (Arindam200) [inline content/bn/virtualization.md:29]:
```suggestion
ভার্চুয়াল মেশিনে চলমান অ্যাপগুলির কোন সচেতনতা নেই, যে তারা একটি ফিজিক্যাল কম্পিউটার ভাগ করছে।
```

File: content/bn/virtualization.md [hunk 2]
```diff
@@ -5,27 +5,28 @@ category: প্রযুক্তি
...
 উদাহরণ হিসেবে, আপনি AWS থেকে একটি "কম্পিউটার" লিজ দিতে পারেন - সেই কম্পিউটারটি আসলে একটি VM।
 
 ## এটা যেসব সমস্যাতে দৃষ্টিপাত করে
 
-ভার্চুয়ালাইজেশন শারীরিক হার্ডওয়্যার ব্যবহারের উন্নতি সহ বেশ কয়েকটি সমস্যার সমাধান করে
-একই ফিজিক্যাল মেশিনে আরও অ্যাপ চালানোর অনুমতি দিয়ে
-নিরাপত্তার জন্য একে অপরের থেকে বিচ্ছিন্ন থাকা অবস্থায়।
+ভার্চুয়ালাইজেশন নিরাপত্তার জন্য একে অপরের থেকে বিচ্ছিন্ন থেকে একই অবস্থায় ফিজিক্যাল মেশিনে
+আরও অ্যাপ চালানোর অনুমতি দেয় এবং 
+শারীরিক হার্ডওয়্যার ব্যবহারের উন্নতি সহ বেশ কয়েকটি সমস্যার সমাধান করে।
```
Comment (Arindam200) [inline content/bn/virtualization.md:24]:
```suggestion
ফিজিক্যাল হার্ডওয়্যার ব্যবহারের উন্নতি সহ বেশ কয়েকটি সমস্যার সমাধান করে।
```

Comment (Debaditya-Som) [reply content/bn/virtualization.md:24]: I have used the term "শারীরিক" it in order to maintain the proper translation of the term "Virtualization" in Bengali.

**Comment by Debaditya-Som** [comment]
@Imtiaz1234 @sajibAdhi kindly review this PR

**Comment by sajibAdhi** [comment]
Hey @Debaditya-Som.

Thank you for your valuable contribution! We generally update Bengali docs in the `dev-bn` branch. Could you kindly change the target branch from `main` to `dev-bn`?

**Comment by Debaditya-Som** [comment]
> Hey @Debaditya-Som.
> 
> Thank you for your valuable contribution! We generally update Bengali docs in the `dev-bn` branch. Could you kindly change the target branch from `main` to `dev-bn`?

yep surely

File: content/bn/virtualization.md [hunk 3]
```diff
@@ -5,25 +5,27 @@ category: প্রযুক্তি
...
 
-## এটা কিভাবে সাহায্য করে
+## এটা কিভাবে সাহায্য করে ?
 
-ভার্চুয়াল মেশিনে চলমান অ্যাপগুলির কোন সচেতনতা নেই যে তারা একটি শারীরিক কম্পিউটার ভাগ করছে।
-ভার্চুয়ালাইজেশন ডেটাসেন্টার ব্যবহারকারীদের মিনিটের মধ্যে একটি নতুন "কম্পিউটার" (ওরফে একটি ভিএম) স্পিন করতে দেয়
-একটি ডেটাসেন্টারে একটি নতুন কম্পিউটার যোগ করার শারীরিক সীমাবদ্ধতার বিষয়ে চিন্তা না করে।
+ভার্চুয়াল মেশিনে চলমান অ্যাপগুলির কোন সচেতনতা নেই, যে তারা একটি ফিজিক্যাল কম্পিউটার ভাগ করছে।
+ভার্চুয়ালাইজেশন ডেটাসেন্টার ব্যবহারকারীদের একটি নতুন "কম্পিউটার" (ওরফে একটি VM) মিনিটের মধ্যে একটি ডেটাসেন্টারে 
+একটি নতুন কম্পিউটার যোগ করার শারীরিক সীমাবদ্ধতার বিষয়ে চিন্তা না করে স্পিন আপ করতে দেয়৷
```
Comment (asem-hamid) [inline content/bn/virtualization.md:30]: একটি নতুন কম্পিউটার যোগ করার ফিজিক্যাল সীমাবদ্ধতার বিষয়ে চিন্তা না করে স্পিন আপ করতে দেয়৷

File: content/bn/virtualization.md [hunk 4]
```diff
@@ -5,25 +5,27 @@ category: প্রযুক্তি
...
-ভার্চুয়ালাইজেশন শারীরিক হার্ডওয়্যার ব্যবহারের উন্নতি সহ বেশ কয়েকটি সমস্যার সমাধান করে
-একই ফিজিক্যাল মেশিনে আরও অ্যাপ চালানোর অনুমতি দিয়ে
-নিরাপত্তার জন্য একে অপরের থেকে বিচ্ছিন্ন থাকা অবস্থায়।
+ভার্চুয়ালাইজেশন নিরাপত্তার জন্য একে অপরের থেকে বিচ্ছিন্ন থেকে একই অবস্থায় ফিজিক্যাল মেশিনে
+আরও অ্যাপ চালানোর অনুমতি দেয় এবং 
+ফিজিক্যাল হার্ডওয়্যার ব্যবহারের উন্নতি সহ বেশ কয়েকটি সমস্যার সমাধান করে।
+
 
-## এটা কিভাবে সাহায্য করে
+## এটা কিভাবে সাহায্য করে ?
```
[the remark this answers is not in the document]
Comment (Debaditya-Som) [reply content/bn/virtualization.md:26]: okay

[the remark this answers is not in the document]
Comment (Debaditya-Som) [reply content/bn/virtualization.md]: I've updated it kindly check it

File: content/bn/virtualization.md [hunk 5]
```diff
@@ -5,25 +5,27 @@ category: প্রযুক্তি
...
 
-## এটা কিভাবে সাহায্য করে
+## এটা কিভাবে সাহায্য করে 
 
-ভার্চুয়াল মেশিনে চলমান অ্যাপগুলির কোন সচেতনতা নেই যে তারা একটি শারীরিক কম্পিউটার ভাগ করছে।
-ভার্চুয়ালাইজেশন ডেটাসেন্টার ব্যবহারকারীদের মিনিটের মধ্যে একটি নতুন "কম্পিউটার" (ওরফে একটি ভিএম) স্পিন করতে দেয়
-একটি ডেটাসেন্টারে একটি নতুন কম্পিউটার যোগ করার শারীরিক সীমাবদ্ধতার বিষয়ে চিন্তা না করে।
+ভার্চুয়াল মেশিনে চলমান অ্যাপগুলির কোন সচেতনতা নেই, যে তারা একটি ফিজিক্যাল কম্পিউটার ভাগ করছে।
+ভার্চুয়ালাইজেশন ডেটাসেন্টার ব্যবহারকারীদের একটি নতুন "কম্পিউটার" (ওরফে একটি VM) মিনিটের মধ্যে একটি ডেটাসেন্টারে 
+একটি নতুন কম্পিউটার যোগ করার শারীরিক সীমাবদ্ধতার বিষয়ে চিন্তা না করে স্পিন আপ করতে দেয়৷
```
Comment (asem-hamid) [inline content/bn/virtualization.md:30]: একটি নতুন কম্পিউটার যোগ করার ফিজিক্যাল সীমাবদ্ধতার বিষয়ে চিন্তা না করে স্পিন আপ করতে দেয়৷

**Comment by asem-hamid** [comment]
@Debaditya-Som do you interested in non code contribution ?

**Comment by Debaditya-Som** [comment]
> @Debaditya-Som do you interested in non code contribution ?

yup i'm interested in contributing whether it's coding or non-coding :)

**Comment by asem-hamid** [comment]
@Debaditya-Som That's great . We have another cncf  localization project  , check it out here

 [Kubernetes website to Bengali](https://github.com/kubernetes/website/issues/31677)

If you interested you can join our [Slack](https://kubernetes.slack.com/messages/kubernetes-docs-bn)

**Comment by Debaditya-Som** [comment]
> @Debaditya-Som That's great . We have another cncf  localization project  , check it out here
> 
>  [Kubernetes website to Bengali](https://github.com/kubernetes/website/issues/31677)
> 
> If you interested you can join our [Slack](https://kubernetes.slack.com/messages/kubernetes-docs-bn)

Yep surely I'm interested in this :)

**Comment by asem-hamid** [comment]
@sajibAdhi  @mitul3737  Please approve this PR

**Review by sajibAdhi — APPROVED** [review]
/lgtm

**Review by mitul3737 — APPROVED** [review]

**Comment by asem-hamid** [comment]
@Debaditya-Som  you can find us here https://kubernetes.slack.com/messages/kubernetes-docs-bn

**Pull request merged**